src/backend/SyncBackendConnection.cpp

```diff
@@ -1043,12 +1068,58 @@ ListModel* SyncBackendConnection::get_model_(
     return parent_sublist->submodel();
 }
 
+std::vector<ListModel*> SyncBackendConnection::get_locator_models_(
+        models::ListModel* parent_model,
+        EntityId id,
+        EntityKind parent_kind)
+{
+    std::vector<ListModel*> models;
+    std::vector<backend::EntityId> parents;
+
+    try
+    {
+        parents = get_entities(parent_kind, id);
```

Comment:
This method is a local one, and so it will never generate an Exception but a void vector

Reply:
Remove try catch clauses